.github/workflows/wolfssl.yml

```diff
@@ -42,7 +42,13 @@ jobs:
 
     - run: ./buildconf && LDFLAGS="-Wl,-rpath,$HOME/wssl/lib" ./configure --enable-warnings --enable-werror ${{ matrix.build.configure }} && make V=1
       name: 'configure and build'
-
+      
+    - run: |
+       sudo apt-get install python-pip
```

Comment:
shouldn't python-pip rather get added to get apt-get install already done above?

Reply:
I see so we can skip this step